api/package.json

```diff
@@ -21,7 +21,7 @@
     "test": "NODE_ENV=test tap -J --cov --coverage-report=lcov --no-browser --no-timeout --reporter=spec --test-arg=--silent '{,!(node_modules)/**/}*.test.js'",
     "test-endpoints": "jest --detectOpenHandles --runInBand '.+\\.endpoint\\.js'",
     "test-specific": "NODE_ENV=test tap --cov --coverage-report=lcov --no-browser --reporter=spec",
-    "prettier": "prettier --single-quote --trailing-comma none --write \"**/*.js\""
+    "prettier": "prettier --single-quote --trailing-comma none --arrow-parens avoid --write \"**/*.js\""
```

Comment:
nice! feel free to add/configure prettier more to suit our needs. would be nice to have it part of the CI/CD at some point too.

Reply:
If we add any more options, we will probably want to stuff them into a `.prettierrc`: https://prettier.io/docs/en/configuration.html

Reply:
there is a prettier config in the web package.json, but we can move it out to it's on config file too